shared/locales/de/donation-certificate.json

```diff
@@ -1,25 +1,11 @@
 {
-	"header": "Verein mit Steuerbefreiung\nim Kanton Zürich",
-	"place": "Zürich,",
-	"country": "Schweiz",
-	"title": "Spendenbescheinigung ",
-	"salutation": "Guten Tag ",
-	"confirmation-1": "Hiermit wird bestätigt, dass Social Income von ",
-	"confirmation-2": ", wohnhaft in ",
-	"confirmation-3": ", im Jahr ",
-	"confirmation-4": " gesamthaft folgende Zuwendungen erhalten hat:",
-	"time-period-1": "Berücksichtigt wurden sämtliche eingegangenen Spenden zwischen dem 01.01.",
-	"time-period-2": " und dem 31.12.",
-	"time-period-3": ".",
-	"contributions-chf": "CHF ",
-	"contributions-eur": "EUR ",
-	"contributions-usd": "USD ",
-	"no-contributions": "Keine Spenden",
-	"information-1": "Bitte beachten Sie für einen möglichen Steuerabzug die kantonalen Regelungen in Ihrem Steuerkanton.",
-	"information-2": "Vielen Dank für Ihre Unterstützung.",
-	"information-3": "Freundliche Grüsse",
-	"signature-1": "Geschäftsführer",
-	"signature-2": "Leiterin Finanzen",
+	"contribution": "{{ amount, currency }}",
+	"email-subject": "Social Income Spendenbescheinigung",
+	"email-body-greeting": "Hallo {{ firstname }}",
+	"email-body-text-1": "Ich sende dir anbei die Spendenbescheinigung für das Jahr {{year}}. Herzlichen Dank für dein Vertrauen und Engagement für weniger Armut und mehr Gerechtigkeit.",
+	"email-body-text-2": "Social Income ist in der Schweiz als gemeinnützige Organisation anerkannt und steuerbefreit.",
+	"email-body-text-3": "Wenn du Fragen hast, dann melde dich einfach bei mir.",
```

Comment:
@ssandino: Shall we maybe replace`melde dich einfach bei mir.` with something like `zögere nicht dich bei mir zu melden?`? `einfach` feels a bit mundart.